docs/src/why-playwright.md

```diff
@@ -35,27 +37,8 @@ Playwright enables fast, reliable and capable automation across all modern brows
 
 * **Capabilities to cover all scenarios**. Support for [file downloads](./downloads.md) and [uploads](./input.md), out-of-process iframes, native [input events](./input.md), and even [dark mode](./emulation.md).
 
-## Integrates with your workflow
-* **One-line installation**. Installing Playwright auto-downloads browser dependencies for your team to be onboarded quickly.
-  ```bash js
-  npm i playwright
-  ```
-  ```bash python
-  pip install playwright
-  playwright install
-  ```
-
-* **TypeScript support**. Playwright ships with built-in types for auto-completion and other benefits.
-
-* **Debugging tools**. Playwright works with the [editor debugger and browser developer tools](./debug.md) to pause execution and inspect the web page.
-
-* **Language bindings**. Playwright is available for [Node.js](https://github.com/microsoft/playwright) [Python](https://github.com/microsoft/playwright-python), [.NET](https://github.com/microsoft/playwright-dotnet) and
-[Java](https://github.com/microsoft/playwright-java). Learn more about [supported languages](./languages.md).
-
-* **Deploy tests to CI**. First-party [Docker image](./docker.md) and [GitHub Actions](./ci.md#github-actions) support to deploy tests to [your preferred CI/CD provider](./ci.md).
```

Comment:
realisation: I see over and over people who didn't know that we have a Docker image.

Reply:
I don't think this page would help them though...